### PR TITLE
Small fixes regarding translations in backend for orders

### PR DESCRIPTION
**Pull request by antonioperic**
…backend with translations

### CODE REVIEW

**Comment by antonioperic** [comment]
It is not problem to rebase, but we dont have shipping.state as payment.state, should we add that?

**Comment by antonioperic** [comment]
There is also some issues, but need answer then will complete it

File: src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/_shippingState.html.twig [hunk 1]
```diff
@@ -13,4 +13,4 @@
     {% set label = 'info' %}
 {% endif %}
 
-<span class="label label-{{ label }}">{{ state|title }}</span>
+<span class="label label-{{ label }}">{{ ('sylius.account.order.shipment.' ~ shipment.state) |trans({'%at%': shipment.updatedAt|date('d.m.Y h:i')}) }}</span>
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/_shippingState.html.twig]: You should not hardcode the date format, pershaps you should use `localizateddate` instead.

**Comment by aRn0D** [comment]
You should squash your commits too.

**Comment by antonioperic** [comment]
@aRn0D no problem for it. Just first to resolve should we add sylius.shipment.state.%s% in translations ?
Same as sylius.payment.state.%s% exist

**Comment by aRn0D** [comment]
if those keys don't exist you need to add them to translation file (the english one).

**Comment by pjedrzejewski** [comment]
We should have just one shipment state translation keys, not separate ones for account. Also you should add these new keys to ``en`` translations, like @aRn0D said. Thanks!

**Comment by michalmarcinkowski** [comment]
Closing as it's outdated (WebBundle is already removed from master).

**Pull request closed without merging**